services/ux-backend/main.go

```diff
@@ -61,9 +86,19 @@ func main() {
 		klog.Info("shutting down!")
 		os.Exit(-1)
 	}
+
+	cl, err := newKubeClient()
+	if err != nil {
+		klog.Errorf("failed to create kubernetes api client: %v", err)
+		klog.Exit("shutting down!")
+	}
+
 	http.HandleFunc("/onboarding-tokens", func(w http.ResponseWriter, r *http.Request) {
 		onboardingtokens.HandleMessage(w, r, config.tokenLifetimeInHours)
 	})
+	http.HandleFunc("/rotate-keys", func(w http.ResponseWriter, r *http.Request) {
+		rotatekeys.HandleMessage(w, r, cl, config.namespace)
+	})
```

Comment:
Why send the namespace and not just have it as an exported global? 
Namespace is a property of the deployment itself, are we expecting different values for dif invocations?

Reply:
Unfortunately I failed to identify the place where this needs to be exported and so ended up passing as a param.

Let me rethink.

Reply:
done.